pkg/source/builtin/cronjob/util.go

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright 2023 The KubeVela Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cronjob
+
+import "github.com/sirupsen/logrus"
+
+var (
+	logger *logrus.Entry
+)
+
+func init() {
+	logger = logrus.WithField("source", cronJobType)
```

Comment:
We can merge the code in `util` and `event` to `crojob.go`.

Reply:
sure, merged